common/domain-setter.sub.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>A page that will likely be same-origin-domain but not same-origin</title>
+
+<script>
+"use strict";
+document.domain = "{{host}}";
```

Comment:
How can document.domain make things cross-origin? I thought it could only relax the restriction.